resources/generate_spec_files.py

```diff
@@ -17,19 +18,29 @@
         spec_dir = Path(f"./resources/spec/{ns['name']}/{ns['version']}")
         spec_dir.mkdir(parents=True, exist_ok=True)
 
+        # create header file
+        header_file = Path(f"./src/spec/{ns['name'].replace('-', '_')}.hpp").resolve()
+        with open(header_file, 'w') as fo:
+            fo.write('#pragma once\n\n')
+            fo.write('#include <string>\n#include <string_view>\n#include <array>\n\n')
+            fo.write(f'namespace AQNWB::spec::{ns['name'].replace('-', '_')}\n{{\n\n')
```

Comment:
```suggestion
            fo.write(f'namespace AQNWB::SPEC::{ns['name'].upper().replace('-', '_')}\n{{\n\n')
```

I believe calling ``upper()`` on the string should be sufficient to make it all uppercase